src/Symfony/Component/Mime/Crypto/SMimeSigner.php

```diff
@@ -30,13 +30,11 @@ final class SMimeSigner extends SMime
     private $privateKeyPassphrase;
 
     /**
-     * @see https://secure.php.net/manual/en/openssl.pkcs7.flags.php
-     *
-     * @param string      $certificate
-     * @param string      $privateKey           A file containing the private key (in PEM format)
+     * @param string      $certificate          The path of the file containing the signing certificate (in PEM format)
```

Comment:
Should the parameter name reflect that this is a path/filename (same for the others)?

Reply:
Probably ... but I don't know if we should make that in this PR.